diff --git a/verilog/dv/aes_test/Makefile b/verilog/dv/aes_test/Makefile
index 82d9c65..199afae 100644
--- a/verilog/dv/aes_test/Makefile
+++ b/verilog/dv/aes_test/Makefile
@@ -30,6 +30,7 @@ include $(MCW_ROOT)/verilog/dv/make/var.makefile
 include $(MCW_ROOT)/verilog/dv/make/cpu.makefile
 
 SOURCE_FILES  := $(SOURCE_FILES) $(shell pwd)/tiny-AES-c_lite/aes.c
+CPUFLAGS := $(CPUFLAGS) -I$(PWDD)/tiny-AES-c_lite
 
 include $(MCW_ROOT)/verilog/dv/make/sim.makefile
 
