diff --git a/litex/caravel.py b/litex/caravel.py
index a95903d..08bcddd 100644
--- a/litex/caravel.py
+++ b/litex/caravel.py
@@ -10,7 +10,6 @@ import argparse
 from os import path
 
 from migen import *
-from litex.soc.cores.spi_flash import SpiFlash
 from litex.soc.integration.soc_core import SoCCore
 from litex.soc.integration.builder import Builder, builder_argdict, builder_args
 from litex.soc.integration.soc_core import soc_core_argdict, soc_core_args
@@ -319,7 +318,7 @@ class MGMTSoC(SoCMini):
 
 def main():
     soc     = MGMTSoC()
-    builder = Builder(soc, compile_software=False)
+    builder = Builder(soc, csr_json="csr.json", compile_software=False)
     builder.build()
 
     lxsocdoc.generate_docs(soc, "build/documentation/", project_name="Caravel Management SoC", author="Efabless")
diff --git a/litex/requirements.txt b/litex/requirements.txt
index 952c0bd..96685cf 100644
--- a/litex/requirements.txt
+++ b/litex/requirements.txt
@@ -5,6 +5,7 @@ git+https://github.com/litex-hub/pythondata-software-picolibc.git
 -e git+http://github.com/m-labs/migen.git#egg=migen
 git+https://github.com/litex-hub/pythondata-cpu-picorv32.git
 git+https://github.com/litex-hub/pythondata-cpu-ibex.git
+git+https://github.com/litex-hub/pythondata-software-compiler_rt.git
 #added to support sim_example
 git+https://github.com/enjoy-digital/litedram
 git+https://github.com/enjoy-digital/liteeth
